pages/tasks/task_details.py

```diff
@@ -14,36 +14,52 @@ class TaskDetailsPage(Base):
 
     _name_locator = (By.CSS_SELECTOR, '.content-container > h1')
 
-    _get_started_button_locator = (By.ID, 'get-started')
-    _save_for_later_button_locator = (By.ID, 'save-for-later')
     _abandon_task_button_locator = (By.ID, 'abandon-task')
     _complete_task_button_locator = (By.ID, 'complete-task')
+    _get_started_button_locator = (By.ID, 'get-started')
+    _save_for_later_button_locator = (By.ID, 'save-for-later')
```

Comment:
For what it's worth, I often order locators based on their position on the page rather than alphabetically. Ultimately I guess I'm saying the sort order doesn't bother me. :)